crates/api_server/src/impls/anvil.rs

```diff
@@ -101,7 +100,9 @@ impl AnvilNamespaceServer for AnvilNamespace {
     }
 
     async fn set_min_gas_price(&self, _gas: U256) -> RpcResult<()> {
-        sh_warn!("Setting minimum gas price is unsupported as ZKsync is a post-EIP1559 chain");
+        tracing::info!(
+            "Setting minimum gas price is unsupported as ZKsync is a post-EIP1559 chain"
+        );
```

Comment:
Well, this was actually better as `sh_warn!` IMHO. Apologies for all these nits 😅

Reply:
No apologies needed sir! Thank you for the review 🙏